### ISSUE TITLE
Option for storing the "poster" coverType as "artist.jpg" in the Artist's folder.

### ISSUE THREAD

**Issue by showipintbri** [Lidarr/Lidarr#5033]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Some music players expect the Artist Image in the Artist folder called artist.jpg (or some definable value). Currently Lidarr pulls the images from fanart.tv and stores them in the `/config/MediaCover/` dir in numbered folders. These numbers correspond to the "id" from the `/api/v1/artist` endpoint.

### Describe the solution you'd like

I'd like a toggle for 'also download images and store them in the Artist folders'.

### Describe alternatives you've considered

I've built a couple bash scripts to be run inside Lidarr which will iterate over all the returned values from the `/api/v1/artist` endpoint, download the referenced 'poster' image and place it in the Artist's folder. After running the script Navidrome and Symphonium (android) instantly pick up and display the correct Artist images.

### Anything else?

```
curl -H "Authorization: Bearer xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" http://127.0.0.1:8686/api/v1/artist > data.txt
for i in $(seq 0 $(($(jq length data.txt)-1))); do

    FILEPATH=$(jq -r .[$i].path data.txt)
    URL=$(jq -r '.['"$i"'].images[] | select(.coverType | test("poster")) | .remoteUrl' data.txt);
    ESCAPE=$(printf "%q" "$FILEPATH")

    # Check if dir exists
    if [ -d ${ESCAPE} ]; then
        # Check if URL isn't empty
        if [ ! -z "$URL" ]; then
            echo "# $FILEPATH"
            echo "# $URL"
            echo ${ESCAPE}
            curl --output "${ESCAPE}/artist.jpg" "$URL"
        fi
    else
        echo "Dir Doesn\'t Exist\: ${ESCAPE}"
    fi

done
```

**Comment by showipintbri** [comment]
I use Navidrome as my streaming music server, and Navidrome isn't looking for `folder.jpg`.

From their [Documentation](https://www.navidrome.org/docs/usage/artwork/#artists):

>Fetching images for artists is controlled by the ArtistArtPriority [config option](https://www.navidrome.org/docs/usage/>configuration-options/#advanced-configuration). This is a comma-separated list of places to look for artist images. The >default is artist.*, album/artist.*, external, meaning:
>
>    - First try to find an artist.* image in the artist folder(s)
>    - If not found, try to find an artist.* image in one of the album folders for this artist
>    - If not found, try to fetch it from an [external service](https://www.navidrome.org/docs/usage/external-integrations) (currently only Spotify)
>    - If not found, use the artist image placeholder (grey star image)

I know the above says it can fetch from Spotify, I don't have a Spotify account, and looking for self-hosted images.

While the page /does/ go on to say "Album" images will use 'folder.jpg' if found, I don't use that method for Album covers(insert SNL joke here). I prefer to embed the Album Cover images in the audio file.

**Comment by showipintbri** [comment]
Yeah, I guess so. A specific "Navidrome" metadata  options aligning the images and locations to those Navidrome would find by default would be perfect.

**Comment by showipintbri** [comment]
I feel like maybe something like this could be easily accomplished at: https://github.com/Lidarr/Lidarr/blob/963ffbea4ec53f1134dcaf8009d4ba3b23952da5/src/NzbDrone.Core/Extras/Metadata/Consumers/Xbmc/XbmcMetadata.cs#L210

Maybe with some clever Find+Replace ;)

**Comment by te5s3rakt** [comment]
> i'm gonna take a stab at this.

any luck?

just rolled my lidarr+tubifarry+slskd+navidrome setup, and artist images are about the last thing I need to solve.
I have a strong preference to saving all my metadata locally, and not only in a services db. got burned one too many times with Plex lol.

**Comment by kyds3k** [comment]
> i'm gonna take a stab at this.

@nerney did you ever make any progress?

**Comment by nerney** [comment]
i did not.
